internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -617,6 +617,12 @@ func (provisioner *KopsProvisioner) ExecMattermostCLI(cluster *model.Cluster, cl
 	return provisioner.ExecClusterInstallationCLI(cluster, clusterInstallation, append([]string{"./bin/mattermost"}, args...)...)
 }
 
+// ExecMattermostCLI invokes the Mattermost CLI for the given cluster installation with the given args.
+func (provisioner *KopsProvisioner) ExecMmctl(cluster *model.Cluster, clusterInstallation *model.ClusterInstallation, args ...string) ([]byte, error) {
+	args = append(args, "--local")
+	return provisioner.ExecClusterInstallationCLI(cluster, clusterInstallation, append([]string{"./bin/mmctl"}, args...)...)
+}
+
```

Comment:
Here's the main addition in the PR.

- I believe we can expect `bin/mmctl` to exist in the CWD, so we can run the command using `./bin/mmctl`
- Passing `--local` at the end of the command will run the command in [Local Mode](https://docs.mattermost.com/manage/mmctl-command-line-tool.html#local-mode), so no authentication credentials are necessary.